BSP/Inc/BSP_SPI.h

```diff
@@ -38,6 +39,9 @@
 #define SPI_GPIOB       0x19
 #define SPI_OLATB       0x1A
 
+// For synchronizing SPI
+extern OS_SEM SPI_Update_Sem4;
```

Comment:
Why does this need to be declared in a visible way? Is something other than BSP_SPI.c going to use it?

Reply:
I just wanted to keep it in the same place as the rest of the semaphores, but reducing visibility sounds like a good idea.